src/client/article.ts

```diff
@@ -222,6 +223,21 @@ function hasSeenCards(): void {
     })
 }
 
+function hydrateQuizAtoms(): void {
+    Array.from(document.querySelectorAll('.js-quiz'))
+        .forEach(atom => {
+            const props = atom.querySelector('.js-quiz-params')?.innerHTML;
+            try {
+                if (props) {
+                    const quizProps = JSON.parse(props.replace(/&quot;/g, '"'));
```

Comment:
I think you can deal with this warning by using `unknown` and checking each field.